setup.py

```diff
@@ -271,14 +271,11 @@ def run(self):
     'cligj>=0.5',
     'click-plugins>=1.0',
     'six>=1.7',
-    'munch']
-
-if sys.version_info < (2, 7):
-    requirements.append('argparse')
-    requirements.append('ordereddict')
-
-if sys.version_info < (3, 4):
-    requirements.append('enum34')
+    'munch',
+    'argparse==1.4.0 ; python_version <= "3.4"',
+    'ordereddict==1.1 ; python_version <= "2.7"',
+    'enum34==1.1.6 ; python_version < "3.4"',
```

Comment:
```suggestion
    'enum34 ; python_version < "3.4"',
```

A library like shapely can't pin its requirements so strictly, it won't be able to coexist with other libraries if it does.

Reply:
@sgillies  thank you for the feedback. I just force-pushed